### PR TITLE
Improves exception message for conflicting ids in stubs.

### PR DESCRIPTION
**Pull request by rjrjr**

### CODE REVIEW

File: workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt [hunk 1]
```diff
@@ -119,7 +119,7 @@ class WorkflowViewStub @JvmOverloads constructor(
 
   override fun setId(@IdRes id: Int) {
     require(id == NO_ID || id != inflatedId) {
-      "id and inflatedId must be distinct"
+      "id must be distinct from inflatedId: ${resources.getResourceName(id)}"
```
Comment (steve-the-edwards) [inline workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt:122]: ${resources.getResourceName(**inflatedId**)

Comment (rjrjr) [reply workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt:122]: `id` is fine, it binds to the method parameter.

Comment (steve-the-edwards) [reply workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt:122]: ahh of course! they are the same :).

**Review by steve-the-edwards — APPROVED** [review]

**Review by zach-klippenstein — APPROVED** [review]
Beautiful.

**Pull request merged**